docs/tutorials/td-tutorial.md

```diff
@@ -0,0 +1,241 @@
+# Send data securely to Arm Treasure Data
+
+There are two ways to send data securely from Mbed OS to Treasure Data:
+
+- HTTPS library - Send data directly to the Treasure Data REST API.
+- Fluentd using fluent logger library - Send data to a hosted Fluentd instance that aggregates and forwards the data on to your treasure data account.
+
+Both libraries are secured with Arm Mbed TLS in transit and are equally secure. We recommend the HTTPS library for development and the Fluentd library for production. The tradeoff between the two is size of code on chip, size of data in transit and setup complexity:
```

Comment:
"Both libraries are secured with Arm Mbed TLS in transit and are equally secure" is a bit clunky, as shown by using "secure" twice

Reply:
Please feel free to reword it.